packages/server/graphql/mutations/endSprintPoker.ts

```diff
@@ -64,13 +64,13 @@ export default {
     if (!currentStageRes) {
       return standardError(new Error('Cannot find facilitator stage'), {userId: viewerId})
     }
+    const {stage} = currentStageRes
+    const phase = getMeetingPhase(phases)
+    stage.isComplete = true
```

Comment:
After these changes, I also did `yarn build`

Reply:
+1 I ran it through the debugger and this part of the fix works. In my test I had 3 stages, completed 1 and ended on 2. However when I stepped into this function, 1 wasn't completed so I ended up with a miscount here.

-1 Please move stage.endAt here as well to keep the stage modifying statements together.